public/locales/kn.json

```diff
@@ -305,44 +305,44 @@
       "Jamtara": "ಜಮ್ತಾರಾ",
       "Khunti": "ಖುಂಟಿ",
       "Lohardaga": "ಲೋಹರ್ದಾಗ",
-      "Ramgarh": "ರಾಮಗ h",
+      "Ramgarh": "ರಾಮಗಢ",
       "Ranchi": "ರಾಂಚಿ",
       "Sahibganj": "ಸಾಹಿಬ್‌ಗಂಜ್",
       "Saraikela Kharsawan": "ಸರೈಕೆಲಾ ಖರ್ಸವಾನ್",
       "Simdega": "ಸಿಮ್ಡೆಗಾ"
     },
     "Karnataka": {
       "title": "ಕರ್ನಾಟಕ",
-      "Chikmagalur": "ಚಿಕ್ಮಗಲೂರ್",
+      "Chikmagalur": "ಚಿಕ್ಮಮಗಳೂರು",
```

Comment:
ಚಿಕ್ಕಮಗಳೂರು ?

Reply:
Thanks @sumanthbhatt. You are correct. Will correct it in my next PR

Reply:
Super; I'll get my setup ready for some Kannada translations as well.

We need the "About/bagge" page also to be translated. I don't see an option in the kn.json file to do it. Is this an item in the list or did I miss something ?

Reply:
The data for About page directly flows from API. The source code for API is available here
https://github.com/covid19india/api

Reply:
Have corrected the spelling for ಚಿಕ್ಕಮಗಳೂರು